### ISSUE TITLE
Invalid option name: 'colorcolumn'

### ISSUE THREAD

**Issue by dhruvmanila** [dhruvmanila/dotfiles#2]
To reproduce:
1. Open Neovim with a new file `nvim test`
2. Enter insert mode with the key `i`

Although, it does not happen using `minimal.vim` with the command `nvim --noplugin -n -u config/nvim/minimal.vim`. It could be something in my config. Need to debug more.

Also, it does not seem to happen when I'm opening Neovim without any arguments, thus opening the dashboard buffer and either entering a session or opening a file through that buffer using telescope.

```
Error detected while processing InsertEnter Autocommands for "*":
E5108: Error executing lua /Users/dhruvmanilawala/.config/nvim/plugin/autocmds.lua:19: Invalid option name: 'colorcolumn'

Error in packer_compiled: ...vim/site/pack/packer/opt/packer.nvim/lua/packer/load.lua:91: Vim(lua):E5108: Error executing lua /Users/dhruvmanilawala/.config/nvim/plugin/autocmds.lua:19: Invalid option name: 'co
lorcolumn'
Please check your config for correctness
```

**Comment by dhruvmanila** [comment]
Related PRs:
- https://github.com/neovim/neovim/pull/14678
- https://github.com/neovim/neovim/pull/14779 (might be fixed by this)

**Comment by dhruvmanila** [comment]
Closed by 708c8e0c18774419d95d29bc5e994cd572c184e5
